src/status_im2/contexts/onboarding/generating_keys/view.cljs

```diff
@@ -54,7 +54,7 @@
                                                            "easing"   (:linear reanimated/easings))))))
   (reanimated/set-shared-value keys-saved-opacity
                                (reanimated/with-delay
-                                4600
+                                5100
```

Comment:
LGTM. Just one question. This number is just by trial right?

Reply:
@BalogunofAfrica i think so, i couldn't find any docs related to these amounts, so i just tried to resolve overlapping between steps